src/fights.ts

```diff
@@ -872,14 +872,19 @@ const freeFightSources = [
               Macro.trySkill($skill`Feel Hatred`).tryItem($item`divine champagne popper`)
             )
             .if_($monsters`giant rubber spider, time-spinner prank`, Macro.kill())
+            .step("pickpocket")
             .skill($skill`Use the Force`)
+            .setAutoAttack();
         );
       }
     },
     false,
     {
       requirements: () => [
-        new Requirement([], { forceEquip: $items`Fourth of May Cosplay Saber` }),
+        new Requirement(["1000 Pickpocket Chance"], {
+            forceEquip: $items`Fourth of May Cosplay Saber`,
+            bonusEquip: new Map<Item, number>([[$item`tiny black hole`, 5000],[$item`mime army infiltration glove`, 5000]]),
```

Comment:
We only want to use these items if we aren't a moxie class; in addition, we only want to equip one of them, not both. Rather than adding this as a bonus equip, I'd add logic to pick one of them as a forceEquip if you have it.

Reply:
Most of the time both will be equipped anyway because they give bonus pickpocket chance that's either better or equivalent to other possibilities.

It's possible that this might not be the case in the future though so I'll try to figure out some logic that makes sense